ios/src/ViewController.mm

```diff
@@ -201,16 +200,18 @@ - (void)setupGL
 {
     [EAGLContext setCurrentContext:self.context];
 
-    Tangram::initialize("scene.yaml");
-    Tangram::loadSceneAsync("scene.yaml");
-    Tangram::setupGL();
+    if (!self.map) {
+        self.map = new Tangram::Map("scene.yaml");
```

Comment:
Yes that's true, I just noticed that we would retain all pointers held by the `Map` class, but it only really depends on how the platform manages process leaks.